### DISCUSSION TITLE
Can I override the build command or output directory in `vercel.json`?

### DISCUSSION THREAD

**Discussion by winston0410** [vercel/vercel#6595] [Help] [answered]
How can I set the build directory in `vercel.json`? I have checked the doc and the option seems missing.

**Comment by leerob** [comment, marked as answer]
You can customize the Build Command for your project in the Project Settings. It's currently not possible to set this command inside your `vercel.json`. However, Vercel should support "zero-conf" for almost any framework you can think of! Could you expand more on what you're trying to override the command for? 

- https://vercel.com/docs/build-step#build-command
- https://vercel.com/docs/build-step#output-directory

**Comment by Ethan-Arrowood** [comment]
Hi @winston0410 ! Great news, we are going to be releasing this exact feature soon. But until then, I'm looking for some existing Vercel users to try it out and provide some feedback. If you would be interested in trying out configuring your build command and output directory via `vercel.json` please send me an email at ethan.arrowood@vercel.com so we can set up some time to chat! Thank you
